src/kudu/consensus/raft_consensus.cc

```diff
@@ -309,6 +309,11 @@ Status RaftConsensus::Init() {
 
   RETURN_NOT_OK(persistent_vars_manager_->LoadPersistentVars(options_.tablet_id, &persistent_vars_));
 
+  if (!persistent_vars_->raft_rpc_token()) {
+    persistent_vars_->set_raft_rpc_token(options_.initial_raft_rpc_token);
+    CHECK_OK(persistent_vars_->Flush());
```

Comment:
are we validating the initial_raft_rpc_token. At some point in time, new rings should enforce this?

Reply:
At some point we can enforce that after this if block we have a token I think. That would make sense.

initial_raft_rpc_token is from topology config, and it might always be empty for the instances we currently have (which will populate directly from persistent_vars after we do a rollout override). We'll have to either wait for them to be replaced or ninja land some kind of json change into topology config